### ISSUE TITLE
Run minimized to tray

### ISSUE THREAD

**Issue by xvybihal** [devqore/myrdp#6]
Hi, is there and option to run minimized in tray?

**Comment by devqore** [comment]
Hi, state of window is remembered.
So if you close the app and main window was hidden then when you start the app next time main window also will be hidden.
Or did you mean something different? Is not possible, at this time, to start application with additional argument which will hide main window.

**Comment by xvybihal** [comment]
Yeah, I though about arguments, it would we nice to tune some things with arguments while starting.
